effectsmanager.h

```diff
@@ -157,6 +150,12 @@ class EffectsManager : public IEffectsManager
                     }
                 }
 
+                auto now = steady_clock::now();
+                while (now < nextFrameTime && _running) {
```

Comment:
I should have commented that the intent is to wait while checking _running every 10ms or so